includes/admin/downloads/metabox.php

```diff
@@ -301,39 +301,54 @@ function edd_render_price_field( $post_id ) {
+				<label for="edd_variable_pricing">
+					<?php echo esc_html( apply_filters( 'edd_variable_pricing_toggle_text', __( 'Enable variable pricing', 'easy-digital-downloads' ) ) ); ?>
+				</label>
+			</div>
+		</div>
 	</p>
 
-	<div id="edd_regular_price_field" class="edd_pricing_fields" <?php echo $price_display; ?>>
+	<div id="edd_regular_price_field" class="edd-form-group edd_pricing_fields" <?php echo $price_display; ?>>
+		<label for="edd_price" class="edd-form-group__label screen-reader-text"><?php esc_html_e( 'Price', 'easy-digital-downloads' ); ?></label>
+		<div class="edd-form-group__control">
 		<?php
 			$price_args = array(
 				'name'  => 'edd_price',
+				'id'    => 'edd_price',
 				'value' => isset( $price ) ? esc_attr( edd_format_amount( $price ) ) : '',
-				'class' => 'edd-price-field'
+				'class' => 'edd-form-group__input edd-price-field',
 			);
-		?>
-
-		<?php if ( $currency_position == 'before' ) : ?>
-			<?php echo edd_currency_filter( '' ); ?>
-			<?php echo EDD()->html->text( $price_args ); ?>
-		<?php else : ?>
-			<?php echo EDD()->html->text( $price_args ); ?>
-			<?php echo edd_currency_filter( '' ); ?>
-		<?php endif; ?>
+			if ( 'before' === $currency_position ) {
+				?>
+				<span class="edd-amount-control__currency is-before"><?php echo esc_html( edd_currency_filter( '' ) ); ?></span>
+				<?php
+				echo EDD()->html->text( $price_args );
+			} else {
+				echo EDD()->html->text( $price_args );
+				?>
+				<span class="edd-amount-control__currency is-after"><?php echo esc_html( edd_currency_filter( '' ) ); ?></span>
+				<?php
+			}
 
-		<?php do_action( 'edd_price_field', $post_id ); ?>
+			do_action( 'edd_price_field', $post_id );
+			?>
+		</div>
 	</div>
 
 	<?php do_action( 'edd_after_price_field', $post_id ); ?>
 
 	<div id="edd_variable_price_fields" class="edd_pricing_fields" <?php echo $variable_display; ?>>
 		<input type="hidden" id="edd_variable_prices" class="edd_variable_prices_name_field" value=""/>
 		<p>
-			<?php echo EDD()->html->checkbox( array( 'name' => '_edd_price_options_mode', 'current' => $single_option_mode ) ); ?>
-			<label for="_edd_price_options_mode"><?php echo apply_filters( 'edd_multi_option_purchase_text', __( 'Enable multi-option purchase mode. Allows multiple price options to be added to your cart at once', 'easy-digital-downloads' ) ); ?></label>
+			<div class="edd-form-group">
+				<div class="edd-form-group__control">
+					<?php echo EDD()->html->checkbox( array( 'name' => '_edd_price_options_mode', 'current' => $single_option_mode, 'class' => 'edd-form-group__input', ) ); ?>
+					<label for="_edd_price_options_mode"><?php echo esc_html( apply_filters( 'edd_multi_option_purchase_text', __( 'Enable multi-option purchase mode. Allows multiple price options to be added to your cart at once', 'easy-digital-downloads' ) ) ); ?></label>
+				</div>
+			</div>
 		</p>
```

Comment:
Can we get rid of the `<p>` tags? I don't think it's valid to have `<div>` tags inside them.